RegexTests/RegexPatternsTest.cs

```diff
@@ -110,5 +110,37 @@ public void InvalidSSNInvalidates()
             bool ssnValidates = Regex.IsMatch(MockConstants.SSNInvalid, SimpleRegex.SocialSecurityNumber);
             Assert.IsFalse(ssnValidates);
         }
+
+        [TestMethod]
+        public void ValidAmbassadorNumberPlate()
+        {
+            bool ambassadorIsValid = Regex.IsMatch(MockConstants.AmbassadorNumberPlateValid, SimpleRegex.DiplomaticNumberPlate);
+            Assert.IsTrue(ambassadorIsValid);
+        }
+
+        [TestMethod]
+        public void InvalidAmbassadorNumberPlate()
+        {
+            bool isBogusAmbassador = Regex.IsMatch(MockConstants.AmbassadorNumberPlateInvalid, SimpleRegex.DiplomaticNumberPlate);
+            Assert.IsFalse(isBogusAmbassador);
```

Comment:
Just one small change here when it comes to variable name..... `bool isBogusAmbassador` here  implies that you are expecting a false then the next line `Assert.isFalse(isBogusAmbassador)` implies a double negative.... maybe changign that to `bool isValidAmbassadorPlate`?

Reply:
Noted. I changed the way I was testing. That's the test that was giving issues because I was testing the wrong thing so before realizing I was passing in an invalid phone number instead of a number plate, I thought to test for the negative !Regex.IsMatch.

Reply:
Cool